src/runtime/pkg/resourcecontrol/cgroups.go

```diff
@@ -41,10 +41,11 @@ func RenameCgroupPath(path string) (string, error) {
 }
 
 type LinuxCgroup struct {
-	cgroup  interface{}
-	path    string
-	cpusets *specs.LinuxCPU
-	devices []specs.LinuxDeviceCgroup
+	sandboxCgroupOnly bool
+	cgroup            interface{}
+	path              string
+	cpusets           *specs.LinuxCPU
+	devices           []specs.LinuxDeviceCgroup
```

Comment:
govet is not happy with the placement of the `sandboxCgroupOnly` field, try the following:
```suggestion
	cgroup            interface{}
	path              string
	cpusets           *specs.LinuxCPU
	devices           []specs.LinuxDeviceCgroup
	sandboxCgroupOnly bool
```

Reply:
@yahaa you'll need to fix the commit message for this commit, see the checks done here: https://github.com/kata-containers/kata-containers/actions/runs/4376736826/jobs/7675992361

Reply:
So it needs a signoff, and a subsystem in the subject line. For the first commit you don't need to copy paste code in, but a description of what going on is helpful.